cwltool/cuda.py

```diff
@@ -0,0 +1,41 @@
+import subprocess  # nosec
+import xml.dom.minidom  # nosec
```

Comment:
```suggestion
import defusedxml.dom.minidom
```
Please and thank you ☺️

Reply:
I saw that in the documentation but didn't want to do it, it means adding another package dependency for questionable security benefit, this is already running an external program that could do lots of things that are more dangerous than returning a malicious XML file.